specification/cognitiveservices/data-plane/ContentSafety/preview/2023-11-30-preview/examples/AnalyzeText.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "Analyze Text",
+  "operationId": "TextOperations_AnalyzeText",
+  "parameters": {
+    "api-version": "2023-11-30-preview",
+    "body": {
+      "text": "This article is over 20 years old, but I think it's especially relevant now given the recent outbreak of the Corona Virus. Israel is Developing 'Ethnic Bomb' for Growing Biological Weapons Arsenal Israel is working on an \"ethnically targeted\" biological weapon that would kill or harm Arabs but not Jews, according to Israeli military and western intelligence sources cited in a front-page report in the London Sunday Times, November 15, 1998"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "blocklistsMatch": [],
+        "categoriesAnalysis": [
+          {
+            "category": "Hate",
+            "severity": 2
+          },
+          {
+            "category": "SelfHarm",
+            "severity": 0
+          },
+          {
+            "category": "Sexual",
+            "severity": 0
+          },
+          {
+            "category": "Violence",
+            "severity": 4
+          }
+        ],
+        "reason": [
```

Comment:
I recommend not include reason in examples for not. Since exposing harmful content will require CELA review

Reply:
fixed.